### ISSUE TITLE
Mondrian Integration

### ISSUE THREAD

**Issue by mailsonsantos** [mekterovic/jqOlapCube#2]
mekterovic, 

on integration with Mondrian do you have any breakthrough?

**Comment by mekterovic** [comment]
No, but on the other hand the term "breakthrough" is a bit exaggerated.
One simply needs to write a small "tunnel" web app to forward MDX queries from jqOlapCube to Mondrian and back (such app is provided for .Net i.e. MS Analysis Services).

If you do write it, please contribute back to the project.

**Comment by mailsonsantos** [comment]
Thanks for the answer, if I write, I'll contribute.
